doc/source/whatsnew/v0.24.1.rst

```diff
@@ -50,7 +50,7 @@ Bug Fixes
 
 **I/O**
 
--
+- Bug in :class:`HDFStore` that caused it to raise ``ValueError`` when reading a Dataframe in Python 3 from dataframe written in Python 2 (:issue:`24925`)
```

Comment:
can you re-word to something like

```
Bug in reading a HDF5 table-format ``DataFrame`` created in Python 2, in Python 3
```

Reply:
am ok with 0.24.1 if you can make this change shortly.

Reply:
doc moved to 0.24.2, reformated to your version. Thanks.